nasa/harmony
# Harmony 1896 - job filtering by label

## Jira Issue ID
HARMONY-1896

## Description
Adds the ability to filter the jobs list using labels

## Local Test Steps
1. Make some Harmony requests with various labels
2. Load the workfow-ui page
3. In the filter input start typing "label". You should a list of labels in the auto-complete (no more than ten)
4. Choose one of the labels.
5. Hit "apply". You should see your jobs only the jobs list only showing jobs with that label.
6. Add another label and hit "apply". You should see jobs that have either of the labels.

## PR Acceptance Checklist
